### ISSUE TITLE
Missing Accessibility support

### ISSUE THREAD

**Issue by ReneVogt** [ReneVogt/WindowsFormsExpander#5]
Add accessibility support to `Expander` control.

**Comment by ReneVogt** [comment]
Accessibility objects have been implemented, but I can't test and verify if this really works out well.
